keyrings/alt/tests/backend.py

```diff
@@ -0,0 +1,170 @@
+# coding: utf-8
```

Comment:
Copying this file from the other project is a bad idea. It means this code will evolve independently and need to have fixes applied to both. Moreover, I can't tell from this commit whether this code was copied exactly as it appears in keyring.testing or if other changes were applied before the commit. Let's find a way to do this without copy/paste code.

Reply:
Oh, I see - much of the duplicated functionality is removed in later commits.